.travis.yml

```diff
@@ -9,8 +9,13 @@ env:
   global:
     - DATABASE_URL='postgres://postgres@localhost/test_utils'
   matrix:
+    - DJANGO='django~=2.0.0' DRF='djangorestframework~=3.11.0'
     - DJANGO='django~=1.11.0' DRF='djangorestframework~=3.9.1'
-    - DJANGO='django~=1.8.0' DRF='djangorestframework~=3.6.0'
+    - DJANGO='django~=1.10.0' DRF='djangorestframework~=3.6.0'
```

Comment:
I would drop Django 1.10 while we're at it. It went out of support in 2017.

Reply:
Once we have a release supporting 1.11 to 2.2, I think we should drop everything below 2.2 and support 2.2, 3.0 and 3.1.